api/Dockerfile-Serdes

```diff
@@ -0,0 +1,38 @@
+ARG BASE_UI_IMAGE=ghcr.io/kafbat/kafka-ui:main
+FROM $BASE_UI_IMAGE
+
+USER root
+RUN mkdir /usr/lib/serdes; \
+    chown -R kafkaui:kafkaui /usr/lib/serdes
+USER kafkaui
+
+# Install Smile Serde
+ARG SERDE_SMILE_JAR=https://github.com/kafbat/ui-serde-smile/releases/download/v1.0.1/serde-smile-v1.0.1-jar-with-dependencies.jar
+ARG SERDE_SMILE_VERSION=1.0.1
+
+RUN cd /usr/lib/serdes; \
+    wget -O "kafka-ui-smile-serde-${SERDE_SMILE_VERSION}.jar" $SERDE_SMILE_JAR
+
+ENV kafka.clusters.0.serde.0.name="Smile"
+ENV kafka.clusters.0.serde.0.filePath="/usr/lib/serdes/kafka-ui-serde-smile-${SERDE_SMILE_VERSION}.jar"
```

Comment:
@kafbat/backend need to discuss this, perhaps we could introduce a way to load serdes from one common path so we don't have to add these many properties and delegate the rest (like serde properties) to the user